acceptance/replication_test.go

```diff
@@ -34,17 +34,17 @@ import (
 )
 
 func countRangeReplicas(client *client.DB) (int, error) {
-	r, err := client.Scan(keys.Meta2Prefix, keys.Meta2Prefix.PrefixEnd(), 10)
+	rng, err := client.Scan(keys.Meta2Prefix, keys.Meta2Prefix.PrefixEnd(), 0)
```

Comment:
Is there any reason not to do an unbounded scan? with 9 nodes some nodes weren't found in the 10-key bound